api/answer.py

```
@@ -239,12 +239,15 @@ def query(self, q_info: dict) -> Optional[str]:
-        # 先过缓存
-        cache_dao = CacheDAO()
-        answer = cache_dao.get_cache(q_info['title'])
-        if answer:
-            logger.info(f"从缓存中获取答案：{q_info['title']} -> {answer}")
-            return answer.strip()
+        # 先过缓存（若存在章节检测错误反馈，说明处于重做模式，跳过缓存让大模型参考反馈重新作答）
+        if not getattr(self, 'work_feedback', None):
+            cache_dao = CacheDAO()
+            answer = cache_dao.get_cache(q_info['title'])
+            if answer:
+                logger.info(f"从缓存中获取答案：{q_info['title']} -> {answer}")
```

Ruff (0.16.2)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)